### ISSUE TITLE
MAINT: Fix failing tests

### ISSUE THREAD

**Issue by prisae** [emsig/empymod#171]
Currently, almost all tests fail. I cannot reproduce the failure locally.

- Linux (except `py3.7 minimal`): `FAILED tests/test_fdesign.py::test_print_data`
- MacOS/Windows: `FAILED tests/test_transform.py::test_hankel[qwe]`

This needs fixing before the next release!

**Comment by prisae** [comment]
```python
import numpy as np


def test_print_data():
    a = 1635.984429995926
    b = 1519.292431702289
    spacing = np.log(a)-np.log(b)
    shift = np.log(a)-spacing*100
    print(np.log(a), np.log(b), spacing, shift)
    assert "1.509903313e-14" == f"{shift:1.10g}"


# Local / until now
# 7.3999999999999995 7.326 0.07399999999999984 1.509903313490213e-14

# New on GHA
# 7.3999999999999995 7.325999999999999 0.07400000000000073 -7.37188088351104e-14
```

**Comment by prisae** [comment]
https://github.com/numpy/numpy/issues/21859

**Comment by prisae** [comment]
Rethinking this: There is a need to improve how `spacing` and `shift` are calculated. Particularly `shift`.

**Comment by prisae** [comment]
Will be fixed in #172
